docsite/rst/dev_guide/repomerge.rst

```diff
@@ -0,0 +1,42 @@
+Repo Merge
+
+Background
+----------
+On Tuesday 6th December 2016, the Ansible Core Team re-merged the module repositories back into `ansible/ansible` in GitHub. The two module repos will be essentially locked, though they will be kept in place for the existing 2.1 and 2.2 dependencies. Once 2.2 moves out of official support (in about a year), we'll completely remove those repositories. Until then, any issues/PRs opened there will be auto-closed with a note to open it on ansible/ansible.
+
```

Comment:
Proposed Edits:
* Removing the language about removing the repositories (we'll lock them but not remove them
* Expand "ansible/ansible" into a full URL so people can click through to the ansible/ansible issue tracker.